v3-docs/v3-migration-docs/breaking-changes/call-x-callAsync.md

```diff
@@ -32,105 +28,209 @@ Meteor.call('getAllData', (err, res) => { // [!code highlight]
+
+If you try to use `Meteor.call` and `Meteor.callAsync` at the same time, like in the examples above, you'll get
+unexpected behavior.
+
+### How you COULD use them together for methods with stub
 
-Meteor.call("someMethod", (err, res) => { // [!code ++]
+Technically, you can use them together for methods with stub if `Meteor.callAsync`
+is called first, and you wait for it to finish:
+
+```js
+import { Meteor } from "meteor/meteor";
+
+await Meteor.callAsync("SOME_ASYNC_METHOD_WITH_STUB")
+  .then((data) => console.log(data))
+  .catch((err) => console.log(err));
+
+// SOME_ASYNC_METHOD_WITHOUT_STUB would work as well
+Meteor.call("SOME_ASYNC_METHOD_WITH_STUB", (err, res) => {
   if (err) {
     console.log(err);
   } else {
     console.log(res);
   }
 });
+```
+
+### How you POSSIBLY can use them without await
 
-await Meteor.callAsync('someMethod') // this is ok // [!code ++]
+If you're calling methods that **DO NOT** have stubs, you can call `Meteor.callAsync` right after a `Meteor.call` and vice-versa:
 
-// this is also ok
-Meteor.callAsync('someMethod').then(data => { // [!code ++]
-  console.log(data);
-  Meteor.callAsync('someMethod').then(data2 => {
-    console.log(data2);
-  });
+```js
+import { Meteor } from "meteor/meteor";
+
+Meteor.callAsync("SOME_ASYNC_METHOD_WITHOUT_STUB")
+  .then((data) => console.log(data))
+  .catch((err) => console.log(err));
+
+Meteor.call("SOME_ASYNC_METHOD_WITHOUT_STUB", (err, res) => {
+  if (err) {
+    console.log(err);
+  } else {
+    console.log(res);
+  }
+});
+```
+
+### How you SHOULD use them
+
+By default, use `Meteor.callAsync`. It'll work for sync and async methods. But if you wish to use `Meteor.call` in some
+specific scenario, use it only when calling sync methods.
+
+Here are some examples:
+
+```js
+import { Meteor } from "meteor/meteor";
+
+// [!code highlight] Example 1: Call serveral Meteor.callAsync in a row.
+// [!code highlight] It doesn't matter if the method has stub or not.
+
+Meteor.callAsync("SOME_ASYNC_METHOD_WITH_STUB");
+Meteor.callAsync("SOME_ASYNC_METHOD_WITHOUT_STUB");
+Meteor.callAsync("SOME_ASYNC_METHOD_WITH_STUB");
+Promise.all([
+  Meteor.callAsync("SOME_ASYNC_METHOD_WITH_STUB"),
+  Meteor.callAsync("SOME_ASYNC_METHOD_WITHOUT_STUB"),
+  Meteor.callAsync("SOME_ASYNC_METHOD_WITH_STUB"),
+]).then(([data1, data2]) => {
+  console.log(data1, data2);
 });
 
+// [!code highlight] Example 2: Call several async methods with `Meteor.callAsync`
+// [!code highlight] and several sync methods with `Meteor.call` in row
 
+Meteor.call("SOME_SYNC_METHOD");
+Meteor.callAsync("SOME_ASYNC_METHOD_WITH_STUB");
+Meteor.call("SOME_SYNC_METHOD");
+Meteor.callAsync("SOME_ASYNC_METHOD_WITH_STUB");
+Meteor.callAsync("SOME_ASYNC_METHOD_WITHOUT_STUB");
 ```
+
+As you can see, if you follow the rule of calling async methods with `Meteor.callAsync` and sync methods with 
+`Meteor.call` you can use them as you want. But ideally, after you finish your migration, you should have any `Meteor.call`
```

Comment:
> ideally, after you finish your migration, you should have any `Meteor.call`

Shouldn't `should` be `shouldn't` here? :) 

Also, I suggest putting this near the top of the "Rules of using Meteor.callAsync & Meteor.call" so it reads something like:

> Technically, `Meteor.callAsync` and `Meteor.call` can be used alongside each other. But ideally, after you finish your migration, you shouldn't use `Meteor.call`, so you don't need to worry about the gotcha scenarios detailed below.